docs/smart-contracts/guides/transactions/invoke-contract-tx-sdk.mdx

```diff
@@ -0,0 +1,358 @@
+---
+title: Invoke a contract function in a Stellar transaction using SDKs
+hide_table_of_contents: true
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+This is a simple example using the Stellar SDK to create, simulate, and then assemble a Stellar transaction which invokes an `increment` function of the [auth example contract](../../example-contracts/auth.mdx).
+
+<Tabs>
+<TabItem value="js" label="JavaScript">
+
+:::tip
+
+Please go to [the project homepage](https://github.com/stellar/js-stellar-sdk) of JavaScript SDK to learn how to install it.
+
+:::
+
+```javascript
```

Comment:
Having some issues with this one. 
```
Sending transaction failed
{"_attributes":{"feeCharged":{"_value":"94770"},"result":{"_switch":{"name":"txBadAuth","value":-6}},"ext":{"_switch":0}}}
```

Are you able to add me on discord? my username is thegodhand_